prepare_zephir_records/run_cid_minter.py

```diff
@@ -0,0 +1,105 @@
+import os
+import sys
+
+import environs
+import json
+import logging
+import time
+
+from lib.utils import db_connect_url
+from lib.utils import get_configs_by_filename
+
+from cid_minter.oclc_lookup import lookup_ocns_from_oclc
+from cid_minter.zephir_cluster_lookup import ZephirDatabase
+from cid_minter.cid_inquiry_by_ocns import cid_inquiry_by_ocns
+from cid_minter.cid_minter import CidMinter 
+
+def convert_comma_separated_str_to_int_list(ocn_str):
```

Comment:
This same function is implemented in `run_cid_inquiry_by_ocns.py`, is there a place it could be implemented that both files can import from?

Reply:
This is a helper function for the command line tool. I will removed it from the cid_inquiry_by_ocns.py script once I have a proper test suit for the run_cid_inquiry_by_ocns.py  script. Thank you.